app/src/main/java/org/simple/clinic/registration/confirmfacility/ConfirmFacilitySheet.kt

```diff
@@ -40,9 +39,22 @@ class ConfirmFacilitySheet : BottomSheetActivity() {
     intent.getSerializableExtra(EXTRA_FACILITY_UUID) as UUID
   }
 
+  private lateinit var binding: SheetRegistrationConfirmFacilityBinding
+
+  private val facilityNameTextView
+    get() = binding.facilityNameTextView
+
+  private val yesButton
+    get() = binding.yesButton
+
+  private val cancelButton
+    get() = binding.cancelButton
+
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    setContentView(R.layout.sheet_registration_confirm_facility)
+
+    binding = SheetRegistrationConfirmFacilityBinding.inflate(layoutInflater)
```

Comment:
We need to clear the binding in accordance with the view lifecycle.

Reply:
We don't need to do that for Activities. Please check the Android documentation about the ViewBinding to get more information about the lifecycle aspects of the ViewBinding.